changelogs/unreleased/gh-7845-httpc-stream-io-interface.md

```diff
@@ -0,0 +1,3 @@
+## feature/httpc
```

Comment:
Nit: Usually `## feature/lua/http client`.

Reply:
Thank you, updated.